appinfo/info.xml

```diff
@@ -5,7 +5,7 @@
 	<name>Mail</name>
 	<summary>IMAP web client</summary>
 	<description>Easy to use email client which connects to your mail server via IMAP and SMTP.</description>
-	<version>0.15.5</version>
+	<version>0.15.6</version>
```

Comment:
O come on... that is worth at least a 0.16

Reply:
well, sure ;)

Reply:
it's a pity we have to include the version in the migration file name. makes it a bit hard to define what the next version will be